idea/src/org/jetbrains/kotlin/idea/intentions/AddBracesIntention.kt

```diff
@@ -76,6 +77,20 @@ class AddBracesIntention : SelfTargetingIntention<KtElement>(KtElement::class.ja
 
         val result = expression.replace(psiFactory.createSingleStatementBlock(expression))
 
+        // Ensure that expression is in single line, due to possible multiple comments
```

Comment:
And yes, you should add tests to this pull request. You need to create some files in `idea\testData\intentions\<YourIntention>` (look at existing examples please), then run "Generate All Tests" and then run `IntentionTestGenerated` to check your tests are passing. Please add tests to this pull request.